components/maincontroller.js

```diff
@@ -195,6 +195,30 @@
             window.subtitleAppearance = data.subtitleAppearance;
         }
 
+        //Report device capabilities
+        if (!window.hasReportedCapabilities) {
+            //Kinda dirty solution but it'll do for now
```

Comment:
```suggestion
            // Kinda dirty solution but it'll do for now
```

Reply:
Maybe explain why it is dirty and what would be better

Reply:
Good point, will do